src/pages/graphql/schema/orders/mutations/request-guest-return.md

```diff
@@ -0,0 +1,144 @@
+---
+title: requestGuestReturn mutation
+edition: ee
+---
+
+# requestGuestReturn mutation
+
```

Comment:
Add the following block below this blank line.

```
<InlineAlert variant="info" slots="text1" />

This mutation is part of the [Storefront Compatibility Package](https://experienceleague.adobe.com/developer/commerce/storefront/setup/storefront-compatibility/). It will be added to Adobe Commerce 2.4.8-beta2.
```